content/en/blog/_posts/2021-XX-XX-writing-a-controller-for-pod-labels.md

```diff
@@ -0,0 +1,473 @@
+---
+layout: blog
+title: "Writing a Controller for Pod Labels"
+date: 2021-03-XX
+slug: writing-a-controller-for-pod-labels
+---
+
+**Authors**: Arthur Busser, Padok
+
+[Operators][what-is-an-operator] are proving to be an excellent solution to
+running stateful distributed applications in Kubernetes. Open Source tools like
+the [Operator SDK][operator-sdk] provide ways to build reliable and maintainable
+operators, and extending Kubernetes to implement custom scheduling needs is now
```

Comment:
```suggestion
operators, making it easier to extend Kubernetes and implement custom scheduling needs.```

Reply:
and remove "is now easier than ever."

Reply:
@rolfedh I took the liberty of fixing https://github.com/kubernetes/website/pull/27010/files#r619728267 to match what i think you intended to suggest.

Reply:
Done 👍